components/eamxx/docs/developer/style/variables.md

```diff
@@ -0,0 +1,30 @@
+# Variables
+
+## Naming
+
+- Please name variables as ***descriptive nouns***.
+    - That is, for a variable holding data related to *horizontal winds*, choose
+      `horizontal_winds` or even `horiz_winds` versus `wh`.
+    - In cases for which this may be untenable, add comments explaining
```

Comment:
Should we add a line with some suggestions on how to mitigate? E.g., don't use articles, or verbs (except `is_`, `can_`, `has_` prefixes, sometimes), look for synonyms, common word truncations (such as `mgr` for manager, or `dev` for device), which can help keep length under control.

Also, how about advocating against removing vowels from words? E.g., we do have `cnt` for `count` in a few places, but a) that doesn't shorten much (and often is not necessary), b) reading the code aloud in a public place can be dangerous, and c) making ppl lose 3 seconds parsing an obscure name cost more than the 1 second it cost to scroll an extra line.